app/models/conference_proceedings_or_journal.rb

```diff
@@ -0,0 +1,15 @@
+# Generated via
+#  `rails generate hyrax:work ConferenceProceedingsOrJournal`
+class ConferenceProceedingsOrJournal < ActiveFedora::Base
+  include ::Hyrax::WorkBehavior
+  include ::ScholarsArchive::ArticleMetadata
+  include ::ScholarsArchive::DefaultMetadata
+  include ScholarsArchive::TriplePoweredProperties::WorkBehavior
+
+  self.indexer = ConferenceProceedingsOrJournalIndexer
```

Comment:
Might be worth considering we eliminate an indexer for each of these models, and make indexers that are composed in a way to handle the mix of metadata that our models use. For instance, this model just needs an indexer that handles indexing both `ArticleMetadata` and `DefaultMetadata`.. whereas other models only need `DefaultMetadata`, and a couple have both `EtdMetadata` and `DefaultMetadata`. I think we could get away with 4 Indexers,

`DefaultIndexer`, `EtdIndexer`, `OerIndexer`, and `ArticleIndexer`.